sample-apps/fdb-profile-analyzer/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.9-slim
+USER root
+
+RUN apt-get update \
+    && apt-get  install -y wget \
+                         bind9-utils \
+                         lsof  \
+                         less \
+                         net-tools \
+                         jq \
+                         openssl \
+                         pkg-config
+
+RUN groupadd -g 4059 foundationdb
+RUN useradd -m -d / -s /bin/bash -u 4059 -o foundationdb -g 4059
+RUN mkdir /app/
+RUN chown foundationdb:foundationdb -R /app/
+RUN pip install foundationdb==7.1.21
```

Comment:
Can we move the `7.1.21` into a build argument for the image to prevent having the same string spread across the Dockerfile?